File: LLD_Java/BookMyShow/Movie/Movie.java
package Java.BookMyShow.Movie;

public class Movie {
    int movieID;
    String movieName;
    int movieDurationInMin;

    public int getMovieID(){
        return movieID;
    }
    public void setMovieID(int movieID){
        this.movieID = movieID;
    }
    public String getMovieName(){
        return movieName;
    }
    public void setMovieName(String movieName){
        this.movieName = movieName;
    }
    public int getMovieDurationInMin(){
        return movieDurationInMin;
    }
    public void setMovieDurationInMin(int movieDurationInMin){
        this.movieDurationInMin = movieDurationInMin;
    }

}

File: LLD(Python)/bookMyShow/Model/Movie.py
class Movie:
    __mid = 0
    def __init__(self):
        self.__movieID = self.__mid
        self.__mid = self.__mid + 1
        self.__movieName = None
        self.__movieDuration = None
        
    def setMovieName(self, name):
        self.__movieName = name

    def getMovieName(self):
        return self.__movieName

    def getMovieID(self):
        return self.__movieID

    def setMovieDuration(self, duration):
        self.__movieDuration = duration

    def getMovieDuration(self):
        return self.__movieDuration




    
        